packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.spec.ts

```diff
@@ -522,83 +480,177 @@ describe('TableColumnDateText', () => {
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is valid and non-zero',
                     data: [{ field: 1708984169258 }],
-                    groupValue: '2/26/2024'
+                    cellValue: '2/26/2024',
+                    groupValue: '2/26/2024',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is incorrect type',
                     data: [{ field: 'not a number' as unknown as number }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is specified and falsey',
                     data: [{ field: 0 }],
-                    groupValue: '1/1/1970'
+                    cellValue: '1/1/1970',
+                    groupValue: '1/1/1970',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is Inf',
                     data: [{ field: Number.POSITIVE_INFINITY }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is -Inf',
                     data: [{ field: Number.NEGATIVE_INFINITY }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is MAX_VALUE',
                     data: [{ field: Number.MAX_VALUE }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is too large for Date',
                     data: [{ field: 8640000000000000 + 1 }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 },
                 {
                     name: 'value is too small for Date',
                     data: [{ field: -8640000000000000 - 1 }],
-                    groupValue: ''
+                    cellValue: '',
+                    groupValue: '',
+                    usesColumnPlaceholder: false
                 }
             ];
 
+            async function initializeColumnAndTable(
+                data: readonly SimpleTableRecord[],
+                placeholder?: string
+            ): Promise<void> {
+                // Set a custom time zone so that the behavior of the test does not
+                // depend on the configuration of the computer running the tests.
```

Comment:
The karma config looks like it is only used for Chrome, so I'm guessing we still want to keep the code in table-column-date-text.spec.ts so that the timezone is correct regardless of what browser is running the tests locally. Does that sound right? Should I still add the timezone to Chrome's karma config and the github action config?

Reply:
Yea we should keep it in the spec.ts 👍
Just trying to do as much as possible to keep environment across CI runs and CI vs local dev consistent (chrome configured the same for local tests in karma and CI configured so if agents run in different timezones)

Reply:
Could also do that in a small standalone PR so CI changes aren't impacting this one